bpf/Makefile

```diff
@@ -88,104 +86,100 @@ $(eval $(call DEFINE_VARIANT,v511))
 $(eval $(call DEFINE_VARIANT,v61))
 
 # ALIGNCHECKER
-objs/%.ll: $(ALIGNCHECKERDIR)%.c
```

Comment:
I find the `.ll` step useful because it allows the inspection of the generated llvm bytecode. Can we keep it? Or at least have a way to generate llvm bytecode. 

That is, now I can  do:
```
make -C bpf objs/bpf_generic_kprobe.ll
llvm-dis bpf/objs/bpf_generic_kprobe.ll -o -
```

Would be nice to keep having something like the above.

Reply:
if it's ok just to add target to generate .ll I'd just add that

Reply:
Yap, whatever works :)